runtime-builder/tests/integration/go-test.yaml

```diff
@@ -1,6 +1,9 @@
 steps:
-- name: 'gcr.io/${PROJECT_ID}/go1-builder:staging'
-- name: 'gcr.io/cloud-builders/docker:latest'
-  args: ['build', '-t', '$_OUTPUT_IMAGE', '.']
-images:
-- '$_OUTPUT_IMAGE'
+- name: gcr.io/gcp-runtimes/integration_test
+  args: [
+    '--no-deploy',
```

Comment:
You'll be able to deploy automatically. Once this is merged and we verify it works you can remove that line.